ccmlib/scylla_node.py

```diff
@@ -1204,17 +1204,17 @@ def _write_agent_log4j_properties(self, agent_dir):
         raise NotImplementedError('ScyllaNode._write_agent_log4j_properties')
 
     def _wait_no_pending_flushes(self, wait_timeout=60):
-        def no_pending_flushes():
-            stdout, _ = self.nodetool('cfstats', timeout=wait_timeout)
+        pending_flushes_re = re.compile(r"^\s+?Pending flushes:\s?(?P<count>\d+)\s?$")
+
+        def no_pending_flushes() -> bool:
+            stdout = self.nodetool('cfstats', timeout=wait_timeout)[0]
```

Comment:
@nyh @bhalevy 
I checked it and it working well
![image](https://user-images.githubusercontent.com/11705430/147654996-96741e18-6dce-4b7c-8063-b3dc7299af5d.png)

Reply:
nit: `\s+?` is an over-complicated equivalent of `\s*`.
I think that this regexp makes more sense: `r"^\s*Pending flushes:\s*(?P<count>\d+)\s*$"`

Reply:
otherwise looks OK

Reply:
You're right, I'm used to doing this but it's wrong in this case.
Fixed.